<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AshuraWin</class>
 <widget class="QMainWindow" name="AshuraWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VEGA</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="inequalitiesGroup">
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Ograniczenia</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>Nierówności</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QScrollArea" name="inequalitiesArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>284</width>
                <height>225</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Przedziały</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableWidget" name="graniceTable">
             <column>
              <property name="text">
               <string>Oznaczenie</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Gr. dolna</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Gr. górna</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Dane wejściowe VEGA</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="solve">
            <property name="text">
             <string>Szukaj rozwiązań</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>pm - prawdopodobieństwo mutacji</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="dwN">
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="dwPC">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="dwPM">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="dwT">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="equalities">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>pc - prawdopodobieństwo krzyżowania</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>N - rozmiar poulacji</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>T - maksymalna liczba pokoleń</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Czas animacji:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="animationTime">
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Tabela Pareto</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="paretoTable"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWidok">
    <property name="title">
     <string>Widok</string>
    </property>
    <addaction name="actionZadania"/>
    <addaction name="action3Dplot"/>
   </widget>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionZapisz_zadanie"/>
    <addaction name="separator"/>
    <addaction name="actionWyjd_2"/>
   </widget>
   <widget class="QMenu" name="menuInformacje">
    <property name="title">
     <string>Informacje</string>
    </property>
    <addaction name="actionAutorzy"/>
    <addaction name="separator"/>
    <addaction name="actionLinki"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuWidok"/>
   <addaction name="menuInformacje"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="Zadania">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Zadania</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionZadania">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zadania</string>
   </property>
  </action>
  <action name="action3Dplot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>plot f2=f(f1)</string>
   </property>
  </action>
  <action name="actionAutorzy">
   <property name="text">
    <string>Autorzy</string>
   </property>
  </action>
  <action name="actionWyjd">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionLinki">
   <property name="text">
    <string>Linki</string>
   </property>
  </action>
  <action name="actionZapisz_zadanie">
   <property name="text">
    <string>Zapisz zadanie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWyjd_2">
   <property name="text">
    <string>Wyjdź</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionZadania</sender>
   <signal>triggered(bool)</signal>
   <receiver>Zadania</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWyjd_2</sender>
   <signal>triggered()</signal>
   <receiver>AshuraWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
